core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -42,13 +42,19 @@ public interface CalciteResource {
   @BaseMessage("''LIMIT start, count'' is not allowed under the current SQL conformance level")
   ExInst<CalciteException> limitStartCountNotAllowed();
 
+  @BaseMessage("''LIMIT start, {0}'' is not allowed under the current SQL conformance level")
+  ExInst<CalciteException> limitStartCountOrAllNotAllowed(String a0);
+
+  @BaseMessage("''OFFSET start LIMIT count'' is not allowed under the current SQL conformance level")
+  ExInst<CalciteException> offsetLimitNotAllowed();
+
   @BaseMessage("APPLY operator is not allowed under the current SQL conformance level")
   ExInst<CalciteException> applyNotAllowed();
 
-  @BaseMessage("JSON path expression must be specified after the JSON value expression")
-  ExInst<CalciteException> jsonPathMustBeSpecified();
```

Comment:
can we retain jsonPathMustBeSpecified() ?

Reply:
Will do suggested changes in next PR